packages/coordinator/src/components/Popup/PopupFlashMessage.tsx

```diff
@@ -0,0 +1,20 @@
+import { Fade } from "@material-ui/core";
+import styles from "./Popup.module.scss";
+
+type PopupFlashMessageProps = {
+  message: string;
+  showFlash: boolean;
```

Comment:
This can work, but I think it would be better to pass `showFlash` and `onHide` props to this component.

In my opinion the timeout should be here and not in the calling class.

Imagine we want to use this flash popup in another place, I don't want to write `setTimeout` twice

Reply:
I partially solved it. now the timeout algorithm is implemented in the component, but still there are two different parameters (message and showFlash). Having it all on the message parameter made a little mess because then the message was empty during the fade out. so in the meanwhile I prefered that it work.